src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/remove-attribute.mdx

```diff
@@ -0,0 +1,544 @@
+---
+title: Remove an attribute
+type: apiDoc
+shortDescription: 'Removes the attribute specified by the string name.'
+tags:
+  - Mobile monitoring
+  - Mobile SDK API
+  - Custom instrumentation
+metaDescription: Mobile app monitoring API used to remove an attribute specified by the string name.
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/removeattribute-android-agent-api
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/removeattribute-android-sdk-api
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/remove-attribute
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/remove-attribute
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/removeattribute-ios-sdk-api
+---
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="android">
+            Android
+        </TabsBarItem>
+        <TabsBarItem id="ios">
+            iOS
+        </TabsBarItem>
+        <TabsBarItem id="capacitor">
+            Capacitor
+        </TabsBarItem>
+        <TabsBarItem id="cordova">
+            Cordova
+        </TabsBarItem>
+        <TabsBarItem id="flutter">
+            Flutter
+        </TabsBarItem>
+        <TabsBarItem id="react">
+            React Native
+        </TabsBarItem>
+        <TabsBarItem id="xamarin">
+            Xamarin
+        </TabsBarItem>
+        <TabsBarItem id="maui">
+            .NET MAUI
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="android">
+## Syntax [#syntax]
+
+### Java [#java]
+
+```java
+NewRelic.removeAttribute(string $name)
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
+
+## Description [#description]
+
+This method removes an attribute specified by the name string. If you want to remove all custom attributes, see [Remove all attributes](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/remove-all-attributes).
+
+For context on how to use this API, see [Send custom attributes and events](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/work-android-sdk-api#attributes-events-insights).
+
+## Parameters [#parameters]
+
+<table>
+  <thead>
+    <tr>
+      <th width={200}>
+        Parameter
+      </th>
+      <th width={200}>
+        Type
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `$name`
+      </td>
+
+            <td>
+        `string`
+      </td>
+
+      <td>
+        Required. The name of the attribute that you want to remove.
+      </td>
+    </tr>
+
+  </tbody>
+</table>
+
+## Return values [#return-values]
+
+Returns `true` if recorded successfully, and `false` if it doesn't.
+
+## Example [#example]
+
+Here's an example of removing an attribute named `rate`:
+
+### Java [#java]
+
+```java
+boolean attributeRemoved = NewRelic.removeAttribute("rate");
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
```

Comment:
@ndesai-newrelic Can you please provide a kotlin example equivalent to the Java example?

Reply:
val attributeRemoved = NewRelic.removeAttribute("rate")

Reply:
```suggestion
```kotlin
    val attributeRemoved = NewRelic.removeAttribute("rate")

```